irods_consortium_continuous_integration_build_hook.py

```diff
@@ -9,16 +9,10 @@
 import irods_python_ci_utilities
 
 
-def add_cmake_to_front_of_path():
-    cmake_path = '/opt/irods-externals/cmake3.21.4-0/bin'
-    os.environ['PATH'] = os.pathsep.join([cmake_path, os.environ['PATH']])
-
 def install_building_dependencies(externals_directory):
     externals_list = [
         'irods-externals-boost1.81.0-2',
-        'irods-externals-clang16.0.6-0',
-        'irods-externals-cmake3.21.4-0',
-        'irods-externals-json3.10.4-0'
```

Comment:
I asked a question about this in the storage tiering repo. See  https://github.com/irods/irods_capability_storage_tiering/pull/349#discussion_r2071530107.